tests/lib/managers/files-test.js

```diff
@@ -2952,4 +2952,144 @@ describe('Files', function() {
 
 	});
 
+	describe('getRepresentationInfo()', function() {
+		var testRepresentation = '[png?dimensions=1024x1024]';
```

Comment:
[style] For constant values like this that don't change between tests, we usually use UPPER_CASE variable names